docs/tables/oci_identity_tenancy.md

```diff
@@ -0,0 +1,31 @@
+# Table: oci_identity_tenancy
```

Comment:
In issue table name is `oci_administration_tenancy_details` and table created is `oci_identity_tenancy` ?

Reply:
@khushboo9024  In terraform data source, getting the like `oci_identity_tenancy`.
https://registry.terraform.io/providers/hashicorp/oci/latest/docs/data-sources/identity_tenancy